frontend/src/reducers/recipeBuilderReducer.ts

```
@@ -431,6 +432,60 @@ export const recipeBuilderReducer = (
         error: null,
       };
 
+    case "UPDATE_UNIT_SYSTEM":
+      const targetSystem = action.payload.unitSystem;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)